docs/high/dataset.rst

```diff
@@ -420,7 +420,7 @@ dataset::
 
 The dtype of the dataset can be accessed via ``<dset>.dtype`` as per normal.
 As empty datasets cannot be sliced, some methods of datasets such as
-``read_direct`` will raise an exception if used on a empty dataset.
+``read_direct`` will raise an **TypeError** exception if used on a empty dataset.
```

Comment:
Maybe use `TypeError` instead which if intersphinx is set up correctly will link to the cpython docs.

Reply:
do you mean link `TypeError` to the cpthon docs?

Reply:
@dota17 
```
``TypeError``
```
or
```
`TypeError`
```
are what you want to replace `**TypeError**` with (I think the first, but maybe the second, I can never remember which one).

Reply:
The first one (double backticks) is an inline code snippet in rst. I'd use that here.

@tacaswell was suggesting that we use a role to link to the definition of `TypeError` in the Python docs. But we don't have intersphinx configured to link to the Python docs at the moment, and I don't think the [description of TypeError](https://docs.python.org/3/library/exceptions.html#TypeError) is that useful to us anyway.